lib/active_admin_import/model.rb

```diff
@@ -132,18 +134,29 @@ def define_methods_for(attr_name)
     end
 
     def encode(data)
-      data = data.force_encoding("UTF-8")
-      unless data.valid_encoding?
-        data = data.encode(force_encoding,
-                           invalid: :replace, undef: :replace, replace: "")
-      end
+      data = content_encode(data) if force_encoding?
+      data = data.encode('UTF-8',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.